server/routes/contacts.js

```diff
@@ -64,18 +64,19 @@ router.post('/', isAuthenticated, (req, res) => {
       email: body.email,
       twitter: body.twitter,
       instagram: body.instagram,
-      github: body.github
+      github: body.github,
+      created_by: body.created_by
```

Comment:
You should use the userId variable from above instead of the body.created_by, that is coming from your user's input and could be anything, whereas the req.user.id, is coming from your own record keeping and usually can't be anything other than the user that's logged in.